analytics.js

```diff
@@ -181,12 +183,23 @@ class Analytics {
         this.loadOnlyIntegrations,
         this.clientIntegrations
       );
+      if (this.options.cookieConsentManager) {
+        this.cookieConsent = new CookieConsentFactory(response, this.options);
+      }
 
       // remove from the list which don't have support yet in SDK
-      this.clientIntegrations = this.clientIntegrations.filter((intg) => {
-        return integrations[intg.name] != undefined;
-      });
-
+      if (this.cookieConsent) {
+        this.clientIntegrations = this.clientIntegrations.filter((intg) => {
```

Comment:
Can you move this inside the filter?

Reply:
done

Reply:
Is this code committed? I don't see the latest updates